OMMatlab.m

```diff
@@ -689,7 +689,7 @@ function simulate(obj,resultfile,simflags)
            
             % support older openmodelica versions before OpenModelica v1.16.2 
             % where linearize() generates "linear_modelname.mo" file
-            if(~isfile(obj.linearfile))
+            if(isfile(obj.linearfile))
```

Comment:
this is still wrong it will not work for the newer openmodelica versions, the condition will always override the obj.linearfile if the people are running with newer openmodelica version

Reply:
@arun3688 Yes bro. This is a mistake on my part to commit. Don't merge this change though.

Reply:
@crazy734, I hope everything is fine, shall we close this issue?

Reply:
@arun3688 Yes. Thank you!